pkg/tnf/test.go

```diff
@@ -135,15 +135,23 @@ func (t *Test) RunAndValidateWithFailureCallback(cb func()) {
 	gomega.Expect(err).To(gomega.BeNil())
 }
 
-// RunWithFailureCallback runs the test, invokes the cb on failure
+// RunWithCallbacks runs the test, invokes the cb on failure/error/success
 // This is useful when the testcase needs to continue whether this test result is success or not
-func (t *Test) RunWithFailureCallback(cb func()) {
+func (t *Test) RunWithCallbacks(scb, fcb func(), ecb func(error)) {
```

Comment:
scb ? ecb ? what do they stand for ?

Reply:
success call back ? error call back ?

Reply:
yes, i can expand the name if it's not clear

Reply:
updated